default/AI/AIDependencies.py

```diff
@@ -88,6 +95,71 @@
 
 PROT_FOCUS_MULTIPLIER = 2.0
 
+# TODO obtain this information from techs.txt
+UNRESEARCHABLE_TECHS = ["SHP_KRILL_SPAWN", "DEF_PLANET_CLOAK"]
+
+UNUSED_TECHS = ["LRN_SPATIAL_DISTORT_GEN", "LRN_GATEWAY_VOID", "LRN_PSY_DOM",
+                "GRO_TERRAFORM", "GRO_BIOTERROR", "GRO_GAIA_TRANS",
+                "PRO_NDIM_ASSMB",
+                "CON_ORGANIC_STRC", "CON_PLANET_DRIVE", "CON_STARGATE",
+                "CON_ART_HEAVENLY", "CON_ART_PLANET",
+                "SHP_NOVA_BOMB", "SHP_DEATH_SPORE", "SHP_BIOTERM"]
+
+THEORY_TECHS = ["LRN_PHYS_BRAIN", "LRN_TRANSLING_THT", "LRN_PSIONICS", "LRN_GRAVITONICS", "LRN_EVERYTHING", "LRN_MIND_VOID", "LRN_NDIM_SUBSPACE", "LRN_TIME_MECH",
+                "GRO_PLANET_ECOL", "GRO_GENETIC_ENG", "GRO_ADV_ECOMAN", "GRO_NANOTECH_MED", "GRO_TRANSORG_SENT",
+                "PRO_NANOTECH_PROD", "PRO_ZERO_GEN",
+                "CON_ASYMP_MATS", "CON_ARCH_PSYCH",
+                "SHP_GAL_EXPLO"]
```

Comment:
Not as simple documentation, or web page explanations, but there is documentation within the code.  The portions of the boost python interface specific to the AI are in /AI/PythonAI.cpp  and more generic portions of the interface are in the files within the  /python folder.  Techs are covered within the Empire wrapper.  There you could see that categories are a tech property, so you can refer to them via ```this_tech.category``` .  Also, Cj has implemented a very helpful tool to assist python IDE's, giving them access to this kind of information, by adding default/AI/freeorion_debug/ide_tools/FreeOrionInterface.py to the python path setting in your IDE.  That way your python IDE would be able to tell you what methods and attributes techs have.  Some of this is available on our wiki at http://www.freeorion.org/index.php/Python_Development, hmm, or should be, it's still much a work-in-progress.   Sorry we didn't make sure you were aware of this IDE extension before.

@Cjkjvfnby when you get a chance could you add some info about using the IDE extension, to http://www.freeorion.org/index.php/AI_Development ? thanks